lua/lvim/utils/functions.lua

```diff
@@ -0,0 +1,17 @@
+local M = {}
+
+M.smart_quit = function()
+  if vim.api.nvim_eval [[&modified]] == 1 then
+    vim.ui.input({
+      prompt = "You have unsaved changes. Quit anyway? (y/n) ",
+    }, function(input)
+      if input == "y" then
+        vim.cmd "q!"
+      end
+    end)
+  else
+    vim.cmd "q!"
```

Comment:
```suggestion
    vim.cmd "q"
```

I'm not sure if there's another reason why `quit` wouldn't continue, other than a modified file, but maybe not risk it?